### PR TITLE
New blog: Joining Decodable

### PR DESCRIPTION
**Pull request by rmoff**
- New blog: Joining Decodable
- Add some more images for future use

### CODE REVIEW

**Comment by rmoff** [comment]
- Videos aren't playing on Chrome, but work fine locally on Chrome and work fine remotely on Safari.

File: content/post/joining-decodable.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+_Well, there's never been any danger of that at any of my previous gigs—I've been fortunate to work with some excellent people—and the same holds true for Decodable_. 
+
+One of the things that really sealed the deal for me during the interview process was just how damn smart the folk were. Not in a "*I ask you stupidly pointless question to highlight how smart I am, you wriggle for an answer that I'll not really listen to, rinse & repeat*" kind of way. But in a focussed and probing and inquisitive way, with true dialogue and listening. That's the kind of stuff you can't LeetCode for, and is a hugely positive indicator. 
+
+As well as being lovely people to chat to and sharp as a sharp thing that's just been sharpened, there's some extensive and deep knowledge about the stream processing world in general and the projects which Decodable uses. These include folk such as [Eric Sammer](https://www.linkedin.com/in/esammer/), [Sharon Xie](https://www.linkedin.com/in/sharonxr/), Flink PMC [Robert Metzger](https://home.apache.org/phonebook.html?uid=rmetzger), and former lead on the [Debezium](https://debezium.io/) project [Gunnar Morling](https://www.morling.dev/).
```
Comment (gunnarmorling) [inline content/post/joining-decodable.md:45]:
```suggestion
As well as being lovely people to chat to and sharp as a sharp thing that's just been sharpened, there's some extensive and deep knowledge about the stream processing world in general and the projects which Decodable uses. These include folk such as [Eric Sammer](https://www.linkedin.com/in/esammer/), [Sharon Xie](https://www.linkedin.com/in/sharonxr/), Flink PMC Chair [Robert Metzger](https://home.apache.org/phonebook.html?uid=rmetzger), and former lead on the [Debezium](https://debezium.io/) project [Gunnar Morling](https://www.morling.dev/). 
```

File: content/post/joining-decodable.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+As well as being lovely people to chat to and sharp as a sharp thing that's just been sharpened, there's some extensive and deep knowledge about the stream processing world in general and the projects which Decodable uses. These include folk such as [Eric Sammer](https://www.linkedin.com/in/esammer/), [Sharon Xie](https://www.linkedin.com/in/sharonxr/), Flink PMC [Robert Metzger](https://home.apache.org/phonebook.html?uid=rmetzger), and former lead on the [Debezium](https://debezium.io/) project [Gunnar Morling](https://www.morling.dev/). 
+
+## What Will I Be Doing at Decodable? 
+
+Who knows?! That's the fun part of joining a 30-person startup 😉
+
+![Conference badge that reads "Robin Moffatt: Chief Shitposter and Meme Artist"](/images/2023/09/IMG_5699.jpeg)
```
Comment (gunnarmorling) [inline content/post/joining-decodable.md:51]: It says "Principal" actually ;)

**Review by gunnarmorling — COMMENTED** [review]
Nice one! Two smalls comments inline. Some of the images in this PR seem not used in this post?

File: content/post/joining-decodable.md (hunk 2 above)
Comment (rmoff) [reply content/post/joining-decodable.md:51]: d'oh! good catch

**Pull request merged**